deployments/kubernetes/monitoring.yaml

```
@@ -0,0 +1,220 @@
+          },
+          {
+            "title": "Audit Log Writes",
+            "targets": [
+              {
+                "expr": "sum(rate(audit_log_writes_total{app=\"admin-api\"}[5m])) by (action)"
+              }
+            ]
+          }
+        ]
+      }
+    }
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)